sections/appendix/metrics.tex

```diff
@@ -16,9 +16,9 @@ \subsection{Topic coherence}\label{app:topic_coherence}
 	S = \{(W', W*)|W' = \{w_i\};w_i \in W;W* = W\}
 \end{equation}
 
-Before calculating word probabilities, a sliding window of size $s$ where $s =110$ is used to create a set of subdocuments $D_s$ over the document set $D$.
+Before calculating word probabilities, a sliding window of size $s$ where $s =110$ is used to create a set of subdocuments $D_s$ over the document set $D$, by creating one subdocument for every window of size $s$ by sliding over each document at a rate of one word per step.
+So document $d = \{w_1, w_2, \dots, w_l\}$, would be converted into $d_1=\{w_1, w_2, \dots, w_s\}, d_2=\{w_2, w_3, \dots, w_{s+1}\}, \dots, d_{l-s}=\{w_{l-s}, w_{1+l-s}, \dots, w_l\}$.
```

Comment:
Hvis et document har en l der er mindre end s, bliver der så kun lavet ét subdokument for dette dokument?

Reply:
ja